### ISSUE TITLE
All-in-one installer

### ISSUE THREAD

**Issue by topper-2k** [PinballY/PinballY-Addons-and-Examples#13]
Along the lines of the Baller Installer, I think PBY needs it's own all-in-one installer. Something that would walk a user through installing VPX, FP, Freezy, Future DMD, Pinup Player and of course PBY. 

Having the option of installing PinAffinity would be a nice touch.

Giving users the option to choose their front-end (Popper, PBX, PBY) would be even better.

I'm sure this is a big ask and probably goes well beyond an "add-on." But it would be awesome. @

**Comment by stephenhouser** [comment]
I had been thinking something along the same lines it would make cabinet setup much easier. I’ve seen the suggestion to use the baller installer and then just install PinballY and disable Pinup Popper. 

I personally have not followed up on the idea due to all the discussion about bundling software from different developers. All these things have different licenses and a project like this should be sure to confirm the ability to bundle with them (or that their license permits it) before including them.

**Comment by topper-2k** [comment]
Yep, agreed about the licenses. I'd assumed Nailbuster (and friends) did that for Freezy, etc. There's another "all in one" out there called Pinball Emporium. Good idea, but unfortunately he bundles a couple hundred tables with his, which is bad form.
